web/src/user_card_popover.js

```diff
@@ -748,8 +710,8 @@ function register_click_handlers() {
         e.preventDefault();
     });
 
-    $("body").on("click", "#recent_view_table .participant_profile", function (e) {
-        const participant_user_id = Number.parseInt($(this).attr("data-user-id"), 10);
+    $("body").on("click", "#recent_view_table .recent_view_participant_avatar", function (e) {
+        const participant_user_id = Number.parseInt($(this).parent().attr("data-user-id"), 10);
```

Comment:
Is it easy to make this change in the click handler target a separate prep commit? I don't understand why it's needed.

Reply:
Sure! Pushed as 5e6f6675.